src/angular-google-analytics.js

```diff
@@ -275,6 +280,9 @@ angular.module('angular-google-analytics', [])
               $window.ga('require', 'ecommerce', 'ecommerce.js');
             } else {
               $window.ga('require', 'ec', 'ec.js');
+              if(curency != 'USD'){
```

Comment:
Is there a reason to make this USD check in the first place? Is there anything wrong with always sending the &cu value even if it contains 'USD' as the currency?